docs/getting-started/pingfederate.md

```diff
@@ -0,0 +1,158 @@
+
+First, start a PingFederate server using Docker. Your DevOps credentials will be read from the `${HOME}/.pingidentity/config` file. The HTTPS port (default `9999`) must be exposed.
+
+```shell
+docker run --name pingfederate_terraform_provider_container \
+  -d -p 9031:9031 \
+  -d -p 9999:9999 \
+  --env-file "${HOME}/.pingidentity/config" \
+  -e SERVER_PROFILE_URL=https://github.com/pingidentity/pingidentity-server-profiles.git \
+  -e SERVER_PROFILE_BRANCH=terraform-provider-pingfederate-1125 \
+  -e SERVER_PROFILE_PATH=terraform-provider-pingfederate/pingfederate \
+  pingidentity/pingfederate:$${PINGFEDERATE_PROVIDER_PRODUCT_VERSION:-12.0.0}-latest
+```
+
+After starting the container, follow the logs until the server becomes available.
+
+```shell
+docker logs -f pingfederate_terraform_provider_container
+```
+
+After you see the following message in the container logs, the server is ready to receive requests from the provider:
+
+```shell
+PingFederate is up
+```
+
+## Accessing the PingFederate API
+
+Using the port mappings from the example above, PingFederate will be available at `https://localhost:9999/pingfederate/app#/`.
+
+If you want to access the API documentation, it is available at `https://localhost:9999/pf-admin-api/api-docs/#/`.
+
+The PingFederate Terraform provider applies configuration at the API endpoints using the `9999` port over HTTPS when accessing the product locally under Docker.
+
+## Determine credentials that are able to configure the server
+
+The configuration API connection established by the provider uses basic authentication. The provider will need the username and password of a user that has permissions to manage server configuration.
+
+In the PingFederate Docker image, the default administrative user has the username `administrator` with password `2FederateM0re`.
+
+## Determine what version of PingFederate you are running
+
+The provider requires that the version of PingFederate is specified.  
+
+You can do this one of two ways:
+
+* Using the `product_version` attribute when configuring the provider:
+
+```shell
+provider "pingfederate" {
+  ...
+  product_version = "12.0.0"
+}
+```
+
+* Setting the `PING_PRODUCT_VERSION` environment variable to the version of PingFederate you are running:
+
+```shell
+export PING_PRODUCT_VERSION=12.0.0
+```
+
+You can view the product version using by running a shell in the container and searching for the appropriate environment variable:
+
+```shell
+docker container exec -it pingfederate_terraform_provider_container /bin/sh
+```
+
+Then, run:
+
+```shell
+env | grep PING_PRODUCT_VERSION
+```
+
+!!! note "Precedence"
+    The `product_version` attribute takes precedence over the `PING_PRODUCT_VERSION` environment variable.
+
+If neither is set, the provider will throw an error.
+
+## Trusting PingFederate certificates
```

Comment:
This section should include info on trusting specific certs using ca_certificate_pem_files